ios/Plugin/PrivacyScreenPlugin.swift

```diff
@@ -33,23 +35,40 @@ public class PrivacyScreenPlugin: CAPPlugin {
     }
 
     @objc func onAppWillResignActive() {
-        guard self.isEnabled else {
-            return
-        }
-        self.privacyViewController = UIViewController()
-        self.privacyViewController!.view.backgroundColor = UIColor.gray
-        self.privacyViewController!.modalPresentationStyle = UIModalPresentationStyle.overFullScreen
-        DispatchQueue.main.async {
-            self.bridge?.viewController?.present(self.privacyViewController!, animated: false, completion: nil)
+            guard self.isEnabled else {
+                return
+            }
+            self.privacyViewController = UIViewController()
+            self.privacyViewController!.view.backgroundColor = UIColor.gray
+            self.privacyViewController!.modalPresentationStyle = UIModalPresentationStyle.overFullScreen
+            DispatchQueue.main.async {
+                self.bridge?.viewController?.present(self.privacyViewController!, animated: false, completion: nil)
+            }
         }
-    }
 
     @objc func onAppDidBecomeActive() {
         DispatchQueue.main.async {
             self.privacyViewController?.dismiss(animated: false, completion: nil)
         }
     }
 
+    @objc private func didDetectRecording() {
+        guard self.isEnabled else {
+            return
+        }
+
+        if #available(iOS 11.0, *) {
+            if UIScreen.main.isCaptured {
+                DispatchQueue.main.async {
+                    self.bridge?.viewController?.present(self.privacyViewController!, animated: false, completion: nil)
```

Comment:
The following code is missing at this point or not?
```swift
self.privacyViewController = UIViewController()
self.privacyViewController!.view.backgroundColor = UIColor.gray
self.privacyViewController!.modalPresentationStyle = UIModalPresentationStyle.overFullScreen
```
(copied from line 41)

Here's what I suggest:
1. New method `presentPrivacyViewController`:
```swift
private func presentPrivacyViewController() {
    self.privacyViewController = UIViewController()
    self.privacyViewController!.view.backgroundColor = UIColor.gray
    self.privacyViewController!.modalPresentationStyle = UIModalPresentationStyle.overFullScreen
    DispatchQueue.main.async {
        self.bridge?.viewController?.present(self.privacyViewController!, animated: false, completion: nil)
    }
}
```
2. New method `dismissPrivacyViewController`:
```swift
private func dismissPrivacyViewController() {
       DispatchQueue.main.async {
            self.privacyViewController?.dismiss(animated: false, completion: nil)
        }
}
```

Use these two methods in `onAppWillResignActive`, `onAppDidBecomeActive` and `didDetectRecording`.